src/lib/pusher.ts

```diff
@@ -13,20 +15,33 @@ const channelName = process.env.NEXT_PUBLIC_PUSHER_CHANNEL!;
 type Payer = {
   lnAddress: string, 
   jackpot: number, 
-  timestamp: number 
+  timestamp: number,
+  eventId: string
 }
 
-export const updateLastPayer = async (lnAddress) => {
-  const amount = parseInt(process.env.INVOICE_AMOUNT ?? '0') || 100;
+export const updateLastPayer = async (lnAddress, event: NostrEvent | null) => {
+  const amount = parseInt(process.env.NEXT_PUBLIC_INVOICE_AMOUNT ?? '0') || 100;
   const previousPayer = await getLastPayer();
   const timeLeft = parseInt(process.env.NEXT_PUBLIC_CLOCK_DURATION ?? '60') - Math.floor((Date.now() - previousPayer.timestamp) / 1000);
   const previousJackpot = timeLeft > 0 ? previousPayer.jackpot : 0;
+  let eventId = previousPayer.eventId
+  if (previousPayer.jackpot === 0 || timeLeft < 0) {
+    // Multiple posts possible if two people start round at same time?
+    // await publishEvent(event)
```

Comment:
Good point... thinking nostr events should be triggered within inngest to avoid double posts. Innget has a feature to "deduplicate" events. I'm using it to prevent double payouts.(https://www.inngest.com/docs/events#deduplication)

Reply:
I played around with this a little. I think it would work with an event like this:
```
  await inngest.send({
    id: `post-${someHash(previousPayer)}`,
    name: 'newRoundPost',
    data: event,
  })
```
  It would also work if we added a round counter into the pusher cache data as well, but that might expire and reset back to 0.
  
  I've been testing this more locally and it seems to work fine. I try to zap twice at the same time from two windows and one creates the post and the other manages to zap it.